dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.facesProvider-4.0.0.Container.feature

```diff
@@ -0,0 +1,9 @@
+# This private feature corresponds to using a JSF-providing feature
+# with just a container and no implementation (e.g. facesContainer-4.0)
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.facesProvider-4.0.0.Container
```

Comment:
Why does the name of this private feature have three version digits?

Reply:
I am guessing that having three digits was supposed to make it look the same as the MyFaces feature? The problem with that is that even if the version of MyFaces was updated, you wouldn't change the name of the feature. I think this version number should be kept in step with the Spec/API/public feature version ... i.e. just `4.0`

Reply:
I've corrected it for faces 4.0 and facesContainer 4.0. Let me know if it looks good, and I can create an issue to follow up on the other jsf/faces issues.

Reply:
Actually, when I tried “io.openliberty.facesProvider-4.0.Container” and “io.openliberty.facesProvider-4.0.MyFaces”, both faces-4.0 and facesContainer 4.0 loaded: 
```

[AUDIT   ] CWWKF0012I: The server installed the following features: [cdi-4.0, expressionLanguage-5.0, faces-4.0, facesContainer-4.0, jndi-1.0, pages-3.1, servlet-6.0, websocket-2.1].
[AUDIT   ] CWWKF0008I: Feature update completed in 1.958 seconds.
```


But my adding changed them to 4.0.0, then the conflict was discovered: 

```
[ERROR   ] CWWKF0033E: The singleton features io.openliberty.facesProvider-4.0.0.Container and io.openliberty.facesProvider-4.0.0.MyFaces cannot be loaded at the same time.  The configured features facesContainer-4.0 and faces-4.0 include one or more features that cause the conflict. Your configuration is not supported; update server.xml to remove incompatible features.
[WARNING ] CWWKF0046W: The configuration includes an incompatible combination of features. As a result, the feature manager did not install any features.
```

Reply:
I'd like to leave this as the 3 digit version for now, but I created a new issue #22787 where I could look into updating it for all the all jsf/faces features.